conf/db/migration/metrics_portal_ddl/common/V3__create_alerts_and_expressions_table.sql

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2015 Groupon
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CREATE TABLE portal.alerts (
+    id SERIAL PRIMARY KEY,
+    version INTEGER NOT NULL DEFAULT 1,
+    created_at TIMESTAMP NOT NULL DEFAULT now(),
+    updated_at TIMESTAMP NOT NULL DEFAULT now(),
+    uuid UUID NOT NULL,
+    name VARCHAR(255) NOT NULL,
+    cluster VARCHAR(255) NOT NULL,
+    service VARCHAR(255) NOT NULL,
+    context VARCHAR(20) NOT NULL CHECK (context IN ('HOST', 'CLUSTER')),
+    metric VARCHAR(255) NOT NULL,
+    statistic VARCHAR(255) NOT NULL,
+    period VARCHAR(30) NOT NULL,
```

Comment:
Still not convinced this should be ISO-8601. Storing it in seconds, e.g. period_in_seconds, as an integer would be more appropriate in this context since complex types like periods not supported as far as I know natively in PG.